modules/assignStyle.ts

```diff
@@ -22,7 +33,7 @@ export default function assignStyle(base, ...extendingStyles) {
         }
 
         if (typeof value === 'object') {
-          base[property] = assignStyle({}, baseValue, value)
+          base[property] = assignStyle({}, baseValue as StyleObject, value)
```

Comment:
This is a good example of the impact made on this library: almost none 🎉! TypeScript only needed a few annotations. Otherwise, **no code was changed.**